java/client/src/main/java/glide/api/commands/SetBaseCommands.java

```diff
@@ -75,4 +75,19 @@ public interface SetBaseCommands {
      * }</pre>
      */
     CompletableFuture<Long> scard(String key);
+
+    /**
+     * Returns whether each member is a member of the set stored at <code>key</code>.
+     *
+     * @see <a href="https://redis.io/commands/smismember/">redis.io</a> for details.
+     * @param key The key of the set to check.
+     * @param members A list of members to check their presence.
```

Comment:
Presence? This is from sismember: 
```suggestion
     * @param members A list of members to check for existence in the set.
```